Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -61,8 +61,10 @@ def check_for_unsupported_macos
         <<~EOS
           You are using macOS #{MacOS.version}.
           #{who} do not provide support for this #{what}.
-          You may encounter build failures or other breakages.
-          Please create pull-requests instead of filing issues.
+          You will encounter build failures and other breakages.
+          Please create pull-requests instead of filing issues or asking the
+          Homebrew community for help. As you are running this #{what} you are
+          the person responsible for resolving your issues (not us).
```

Comment:
`your issues` → `any issues` to avoid discussions like _“this is clearly $SOFTWARE’s fault so it’s actually not **my** issue”_